app/components/wallet/add/option-dialog/OptionBlock.js

```diff
@@ -12,7 +13,7 @@ import styles from './OptionBlock.scss';
 type Props = {|
   parentName: string,
   type: string,
-  title: string,
+  title: string | Element<any>,
```

Comment:
can we use `FormattedHTMLMessage` over here?

Reply:
I tried it without success
```
```Cannot create OptionBlock element because React.Element [1] is incompatible with FormattedHTMLMessage [2] in property
title.

     app/components/wallet/add/option-dialog/WalletRestoreOptionDialog.js
     62│           <ul className={styles.optionBlockList}>
     63│             <OptionBlock
     64│               parentName="WalletRestoreOptionDialog"
     65│               type="restoreNormalWallet"
 [1] 66│               title={<FormattedHTMLMessage {...messages.restoreNormalTitle} />}
     67│               learnMoreText={intl.formatMessage(messages.restoreNormalDescription)}
     68│               onSubmit={onRestore}
     69│             />
     70│             <OptionBlock
     71│               parentName="WalletRestoreOptionDialog"

     app/components/wallet/add/option-dialog/OptionBlock.js
 [2] 17│   title: string | FormattedHTMLMessage,```
```

Reply:
@YankoFelipe 

how about? 
`title: string | Element<any>` -> `title: string`

I guess `FormattedHTMLMessage` will evaluate to `string` type.

Reply:
```
Cannot create OptionBlock element because React.Element [1] is incompatible with string [2] in property title.
     app/components/wallet/add/option-dialog/WalletRestoreOptionDialog.js
     60│       >
     61│         <div className={styles.component}>
     62│           <ul className={styles.optionBlockList}>
     63│             <OptionBlock
     64│               parentName="WalletRestoreOptionDialog"
     65│               type="restoreNormalWallet"
 [1] 66│               title={<FormattedHTMLMessage {...messages.restoreNormalTitle} />}
     67│               learnMoreText={intl.formatMessage(messages.restoreNormalDescription)}
     68│               onSubmit={onRestore}
     69│             />
     70│             <OptionBlock
     71│               parentName="WalletRestoreOptionDialog"
     72│               type="restorePaperWallet"

     app/components/wallet/add/option-dialog/OptionBlock.js
 [2] 16│   title: string,
```